src/Slot-Tests/LazyClassVariableTest.class.st

```diff
@@ -39,7 +39,7 @@ LazyClassVariableTest >> testLazyClassVarReadReflective [
 { #category : #tests }
 LazyClassVariableTest >> testLazyClassVarReadReflectiveBlock [
 	| classVar |
-	classVar := #ClassVar => LazyClassVariable defaultBlock: [5].
+	classVar := #ClassVar => LazyClassVariable default: [5].
 	aClass := self make: [ :builder | builder sharedVariables: {classVar}].
```

Comment:
Side question. I noticed that class builder uses name sharedVariable. Do we have a plan to rename ClassVariable to SharedVariable?

Reply:
We use the term "shared variable" *only* in the API if the class builder.

Thus tried to do the opposite and change the API of the classbuilder, but it broke the bootstrap. Need to look at it again.